File: pset6/prob/mario.py
from cs50 import get_int

h = 0

# get height
while h < 1 or h > 8:
    h = get_int("Height: ")

# print spaces and hashes
for i in range(h):
    for j in range(h):
        if i + j < h - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print("")

File: pset1/prob/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompt for height

    int n;

    do
    {
        n = get_int("Height: ");
    }

    while (n < 1 || n > 8);

    // dots (change to spaces) and hash

    n = n + 1 ;

    int i;
    int j;

    // vertical condition
    for (i = 1; i < n; i++)
    {
        // horizontal condition
        for (j = 0; j < n - 1; j++)
        {
            // condition for spaces
            if (i + j < n - 1)

                printf(" ");

            else
                printf("#");
        }

        printf("\n");
    }

}